src/librustc_mir/transform/qualify_consts.rs

```diff
@@ -101,7 +101,9 @@ struct Qualifier<'a, 'gcx: 'a+'tcx, 'tcx: 'a> {
     rpo: ReversePostorder<'a, 'tcx>,
     tcx: TyCtxt<'a, 'gcx, 'tcx>,
     param_env: ty::ParamEnv<'tcx>,
-    local_qualif: IndexVec<Local, Option<Qualif>>,
```

Comment:
The commit message isn't up to date anymore. The temp + return merge is already in master